resources/views/faset/faset.blade.php

```diff
@@ -67,6 +67,13 @@
               <input type="checkbox" class="custom-control-input" id="heardfrom-vex" name="heardfrom" value="vex">
               <label class="custom-control-label" for="heardfrom-vex">Vex Event</label>
             </div>
+            <div class="custom-control custom-checkbox">
+              <input type="checkbox" class="custom-control-input" id="heardfrom-other" name="heardfrom" value="other">
```

Comment:
Make the Other box only required when the box is checked, otherwise it should be disabled. Here's a helpful snippet from the prototype FASET form:

https://github.com/RoboJackets/apiary/blob/a27231fda6f994e08669748c0b034f8315058e40/resources/views/faset.blade.php#L52-L56